test/system/700-play.bats

```diff
@@ -419,11 +419,17 @@ _EOF
     # Force removal of container
     run_podman rm --force -t0 $PODCTRNAME
     # Create external container using buildah with same name
-    buildah from --name $PODCTRNAME $IMAGE
+    echo "$_LOG_PROMPT buildah from --name $PODCTRNAME $IMAGE"
+    run buildah from --name $PODCTRNAME $IMAGE
+    echo "$output"
+    buildah_ctr="$output"
+    assert "$status" -eq 0 "buildah from"
+
     # --replace deletes the buildah container and replace it with new one
     run_podman play kube --replace $TESTYAML
```

Comment:
The comment claims it deletes the buildah container, if this doesn't work then the test seems pointless overall and podman doesn't work correctly!?

Reply:
That's not what the test was testing. From my reading of the test (which is quite possibly wrong) and the [PR comment log](https://github.com/containers/podman/pull/20457#issuecomment-1785222664) (ditto), it's the `--replace` that would fail?

Reply:
My reading is without --replace this should fail as the name is already in use by buildah.
And if it is actually replacing then the buildah ctr must be removed after the play kube command so no leaks should happen. I don't see a world where podman and buildah could actually use the same name.

Looking more at this using the pod name seems wrong here because that is never used as storage name. IT must use a ctr name.

Reply:
`$PODCTRNAME` is a container name. It's the name of the container inside the pod.

I seem to have broken this test with my `safename` PR. At the time #20025 merged, the new test failed on (then) main and passed on the PR. Now, the `--replace external` test passes when run against podman @ 91df369^^ . The offending commit is 48aea083c093d3e9520c0ea2042036c7166bd6f4, and the cause was that `_write_yaml()` stopped creating a default container.  Fix in progress.